src/open_inwoner/configurations/constants.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+
+class FontFileName(models.TextChoices):
+    body = _("text_body_font"), _("Text body font")
+    heading = _("heading_font"), _("Heading font")
```

Comment:
Do we need this file? The same choices are defined just above this in ./choices.py